src/components/callBackFunctions/IsIncludes.tsx

```diff
@@ -0,0 +1,30 @@
+import { Todo } from '../../types/Todo';
+
+const filteredTodos = (
+  todos: Todo[],
+  query: string,
+  filterBy: string,
+) => {
+  let filteredArray = [...todos];
+
+  if (query) {
+    filteredArray = filteredArray
+      .filter(item => item.title.toLowerCase().trim()
+        .includes(query.toLowerCase().trim()));
+  }
+
+  switch (filterBy) {
+    case 'Completed':
```

Comment:
Completed, Active must be enum for prevent typo error, because you use it in different files